assemblies/assembly-configuring-the-global-header.adoc

```diff
@@ -0,0 +1,19 @@
+:_mod-docs-content-type: ASSEMBLY
+:context: configuring-the-global-header-in-rhdh
+[id="{context}"]
+= Configuring the global header in {product}
+
+As an administrator, you can configure the {product} global header to create a consistent and flexible navigation bar across your {product-short} instance.
+By default, the {product} global header includes the following:
+
+* Search input field
+* Create button linking to software templates
+* Support for internal and external links, for example, support and notification icons
+* Notification buttons
+* User profile and create drop-down
```

Comment:
There is no create dropdown in 1.5. There is just the create button that you have mentioned as bullet point 2

```suggestion
* User profile drop-down
```